src/main.js

```diff
@@ -8,25 +8,19 @@ import "vue3-circle-progress/dist/circle-progress.css";
 import * as Sentry from "@sentry/vue";
 import { Integrations } from "@sentry/tracing";
 import { sentry as sentryConfig } from "./utils/constants";
-import { Buffer } from "buffer";
-import { Stream } from "stream-browserify";
-
-window.global = window;
-window.Buffer = window.Buffer || Buffer;
-window.Stream = Stream;
 
 const app = createApp(App);
-// Sentry.init({
-//   app,
-//   dsn: sentryConfig.dsn,
-//   integrations: [
-//     new Integrations.BrowserTracing({
-//       routingInstrumentation: Sentry.vueRouterInstrumentation(router),
-//       tracingOrigins: sentryConfig.tracingOrigins,
-//     }),
-//   ],
-//   tracesSampleRate: 1.0,
-// });
+Sentry.init({
+  app,
+  dsn: sentryConfig.dsn,
```

Comment:
The dsn value should default to `null` during `development`. Can we add a condition in `./utils/constants.js`?

[See this](https://github.com/getsentry/sentry-java/issues/574).

Reply:
Check the updated code and let me know if this is fine